.gitignore

```diff
@@ -5,3 +5,4 @@ out/
 cache/
 node_modules/
 .vscode/
+remappings.txt
```

Comment:
VsCode requires `remappings.txt` to be able to map imports in Solidity files, that's the main reason why I put it there.
In case if we need custom remappings we can remove it from gitignore